docs/cmd-deps-convert.md

```diff
@@ -1,8 +1,15 @@
 # dephell deps convert
 
-Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts. Dependency resolution is the strongest side of the DepHell, pip and pipenv has no any dependency resolution that would be smart enough.
+Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts.
 
-You can specify input and output in three different ways:
+Dephell uses four pieces of information for conversion:
```

Comment:
Most of the markdown parsers (for example, `blackfriday` in Go, gitlab, or bitbucket) can't parse lists if they aren't prepended by empty lines. Please, add an empty line after this sentence to be sure that it will be rendered correctly.

Reply:
Good to know, thanks.  Will fix